<?xml version="1.0" ?>
<robot name="dron08">
    <webots>
        <device reference="camera" type="Camera">
            <ros>
                <alwaysOn>false</alwaysOn>
                <enabled>false</enabled>
                <update_rate>10</update_rate>
            </ros>
        </device>
        <plugin type="webots_ros2_driver::Ros2IMU">
            <enabled>true</enabled>
            <topicName>imu</topicName>
            <alwaysOn>true</alwaysOn>
            <frameName>base_link/imu_link</frameName>
            <inertialUnitName>inertial unit</inertialUnitName>
            <gyroName>gyro</gyroName>
        </plugin>
        <plugin type="uned_crazyflie_webots.crazyflie_driver.CrazyflieWebotsDriver">
            <name_id>dron08</name_id>
            <config_file>my_config_file</config_file>
        </plugin>
    </webots>
</robot>